AUTHORS.rst

```diff
@@ -0,0 +1,9 @@
+The Box Python SDK is an open source project supported by `Box <https://box.com>`_
+used to interact with the Box API. This is a list of contributors.
+
+- `@Jeff-Meadows <https://github.com/Jeff-Meadows>`_
+- `@jmoldow <https://github.com/jmoldow>`_
+- `@aptxkid <https://github.com/aptxkid>`_
+- `@hnguyen08 <https://github.com/hnguyen08>`_
+- `@potrebic <https://github.com/potrebic>`_
+- `@nsundareswaran <https://github.com/nsundareswaran>`_
```

Comment:
I added those who are part of the Box org; additional contributors should feel free to add themselves, but I'd rather not add without their permission. Full names should be fine - anyone who wants that should make that change.